java/client/src/test/java/glide/managers/CommandManagerTest.java

```diff
@@ -107,32 +109,56 @@ public void submitNewCommand_return_String_result() {
         // exercise
         CompletableFuture<Object> result =
                 service.submitNewCommand(
-                        command, new BaseCommandResponseResolver((p) -> p == pointer ? testString : null));
+                        CustomCommand,
+                        new String[0],
+                        Optional.empty(),
+                        new BaseCommandResponseResolver((p) -> p == pointer ? testString : null));
         Object respPointer = result.get();
 
         // verify
         assertTrue(respPointer instanceof String);
         assertEquals(testString, respPointer);
     }
 
+    @SneakyThrows
+    @Test
+    public void submitNewCommand_throws_ClosingException() {
```

Comment:
Move to exception handling if it is not a duplicate

Reply:
Removing.  This test is obsolete.